index.js

```diff
@@ -21,5 +25,28 @@ addTaskButton.addEventListener("click", () => {
     createdAt: new Date().toISOString(),
   });
 
-  console.log(tasks);
+  tasks.forEach((task) => {
+    const taskCreatedAt = Object.values(task)[3]
+      .toString()
+      .match(/\d\d:\d\d:\d\d/);
```

Comment:
Why not? Please read `forEach` syntax on mdn.
```js
const taskCreatedAt = task.createdAt .match(/\d\d:\d\d:\d\d/);
```

Reply:
Also, `createdAt` is string. there is no need to convert it to string.

Reply:
yeah that's true,thanks.